mitodl/open-discussions
# Refactor betamax cassette code to automatically create cassettes

#### What are the relevant tickets?
Fixes #304 
Copied from `nl/easy_betamax_cassettes`

#### What's this PR do?
Fixes configuration of fixtures to allow creation of betamax cassettes when tests are run without an existing cassette.

#### How should this be manually tested?
Go to `channels/views_test.py` `test_create_channel`. Alter the username in `UserFactory.create()` to a username you use to log into reddit. Alter the channel details to your liking, making sure that the channel name does not already exist.

Delete `channels.views_test.test_create_channel.json` and run the test. The test should pass. Make sure the cassette file was created and is not empty. Run the test again and it should pass again.
